src/views/Roster.vue

```diff
@@ -142,6 +152,7 @@ export default {
   data: () => ({
     error: undefined,
     isDownloading: false,
+    photosPerPage: false,
```

Comment:
@mlincoln - I would name this variable same as suggested before: `showOnePhotoPerPage`